5-Cookies/client-react/src/components/TodoList/TodoListItem/TodoListItem.spec.tsx

```diff
@@ -0,0 +1,53 @@
+import { mockTodo } from "../../../test/utils";
+import { TodoListItemDriver } from "./TodoListItem.driver";
+
+describe("todo list item component", () => {
+  let driver : TodoListItemDriver;
+
+  beforeEach(() => {
+    driver = new TodoListItemDriver();
+  });
+
+  test("should display description text", async () => {
+    const item = { text: "test" };
+    driver.given.item(item);
+
+    driver.when.render();
+
+    expect(driver.get.todoText()).toEqual(item.text);
+  });
+
+
+  test("should display switchMode status", async () => {
+    driver.when.render();
+
+    expect(driver.get.todoStatus()).toEqual(false);
```

Comment:
why false? at least describe the expectation of the product in the test description

Reply:
this test has been removed